dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -1373,6 +1384,10 @@ private String getStringValue(Map<String, Object> row, String key) {
 		Object value = row.get(key);
 		return (value == null) ? "" : value.toString();
 	}
+
+	private Long getLongValue(final Map<String, Object> row, final String key) {
+		return ConversionUtils.toLong(row.get(key), 0l);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Upper-case this literal "l" suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS818)